daemonize.go

```diff
@@ -152,7 +152,7 @@ func Run(
 	path string,
 	args []string,
 	env []string,
-	status io.Writer) (err error) {
+	status io.Writer, stderr *os.File) (err error) {
```

Comment:
Please put this argument on its own line, like the others.

Reply:
Done